src/tools.h

```diff
@@ -55,7 +55,7 @@ namespace zim {
                                 const std::string & delims=" *-");
 
   std::map<std::string, int> read_valuesmap(const std::string& s);
-
+  void setICUDataDirectory(const std::string& path);
```

Comment:
setICUDataDirectory is already (publicly) declared in (`zim/tools.h`)[https://github.com/openzim/libzim/blob/master/include/zim/tools.h#L35].

So we don't need to declare it here.

Reply:
done